editor/resources/templates/template.cp

```diff
@@ -0,0 +1,13 @@
+#version 430
+
+layout (local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
+
+layout(rgba32f) uniform image2D texture_out;
+
+uniform vec4 color;
+
+void main()
+{
+    ivec2 tex_coord = ivec2(gl_GlobalInvocationID.xy);
+    imageStore(texture_out, tex_coord, color);
+}
```

Comment:
Maybe the template should just be this?

```
#version 430

layout (local_size_x = 1, local_size_y = 1, local_size_z = 1) in;

void main()
{
}
```

It's kinda like the "function init(self)" snippets in a script, it's not up to us to decide for you what you are going to do. There's no barebones example of what a compute shader should do, it totally depends on the usecase.